platform/kubernetes/launch.go

```diff
@@ -0,0 +1,269 @@
+package main
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"log"
+	"net"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/sipb/homeworld/platform/keysystem/worldconfig/paths"
+	"github.com/sipb/homeworld/platform/kubernetes/wrapper"
+)
+
+func kubeConfigPath(suffix string) string {
+	return "/etc/homeworld/config/kubeconfig-" + suffix
+}
+
+func LaunchAPIServer() error {
+	clusterConf, err := wrapper.GetClusterConf()
+	if err != nil {
+		return errors.Wrap(err, "while loading cluster.conf")
+	}
+	localConf, err := wrapper.GetLocalConf()
+	if err != nil {
+		return errors.Wrap(err, "while loading local.conf")
+	}
+	// parse the keys before using them for the sake of validation
+	apiserverCount := clusterConf["APISERVER_COUNT"]
+	if _, err := strconv.ParseUint(apiserverCount, 10, 32); err != nil {
+		return errors.Wrap(err, "while validating APISERVER_COUNT")
+	}
+	serviceCidr := clusterConf["SERVICE_CIDR"]
+	if _, _, err := net.ParseCIDR(serviceCidr); err != nil {
+		return errors.Wrap(err, "while parsing SERVICE_CIDR")
+	}
+	hostIP := localConf["HOST_IP"]
+	if net.ParseIP(hostIP) == nil {
+		return fmt.Errorf("invalid IP '%s'", hostIP)
+	}
+	etcdEndpoints := clusterConf["ETCD_ENDPOINTS"]
+	if strings.Count(etcdEndpoints, "://") < 1 { // just a sniff test, because the format is complex
+		return fmt.Errorf("missing or invalid ETCD_ENDPOINTS: '%s'", etcdEndpoints)
+	}
+	cmd := exec.Command(
+		"/usr/bin/hyperkube", "kube-apiserver",
+		// role-based access control
+		"--authorization-mode", "AlwaysAllow",
+		// number of api servers
+		"--apiserver-count", apiserverCount,
+		// public addresses
+		"--bind-address", "0.0.0.0", "--advertise-address", hostIP,
+		// IP range
+		"--service-cluster-ip-range", serviceCidr,
+		// use standard HTTPS port for secure port
+		"--secure-port", "443",
+		// etcd cluster to use
+		"--etcd-servers", etcdEndpoints,
+		// don't allow privileged containers: don't allow this kind of thing
+		"--allow-privileged", "true",
```

Comment:
what does this comment mean?

Reply:
Looking at the history, it comes from the very earliest versions of Hyades back in 2017, got contradicted in meaning when allow-privileged changed to true without changing the comment, and now has no real meaning. I'm going to eliminate it.